### ISSUE TITLE
create assoc logo

### ISSUE THREAD

**Issue by skippy** [sjifire/website#342]
it needs to be different than our dept logo.  In the past, they took the dept logo, stripped out Fire Dept, and then put assoc in big white letters around the bottom half.  Let's see if we can do better.   thoughts:

- Change bottom 'Fire Dept' to Association?
- remove red in the maltese and make it a different color (or transparent with just a black outline?)

what else?

**Comment by icegulch** [comment]
Vectorized initial concept...


[SJI-FIRE-ASSN-v1.pdf](https://github.com/sjifire/website/files/10701569/SJI-FIRE-ASSN-v1.pdf)

**Comment by skippy** [comment]
Would you mind creating a few versions so we can see the spacing constraints?  Such as 'fire assoc' and 'ffa' in the bottom section of the cross?

**Comment by icegulch** [comment]
Of course! To be clear, this was a quick concept. I noticed the "Florian Cross" I obtained from wikimedia has jagged edges, which is kind of a bummer. But, we can clean that up. I've seen other departments (and associations) place the logo inside a "shield" (badge) and put text around that, so I can try that too. I'll riff on the ocean a bit too... Pretty strapped through Tuesday, so don't panic if you don't hear from me!

**Comment by icegulch** [comment]
With all letters the same type size but the bottom type kerned more tightly (tighter letter spacing). Happy to riff from here. Like I said, we could use the same wording as the official logo, but then place the emblem on a different badge / patch shape with wording to indicate "Association" (spelled out). Am I explaining that well?

https://www.augustachronicle.com/story/news/local/columbia-county-news-times/2013/02/06/patches-carry-new-name-of-fire-department/14191975007/

[SJI-FIRE-ASSN-v2.pdf](https://github.com/sjifire/website/files/10717260/SJI-FIRE-ASSN-v2.pdf)

**Comment by icegulch** [comment]
Oh, and I cleaned up the Florian Cross (just re-created it as a nice clean-edge vector)...

**Comment by skippy** [comment]
thanks; I'll take this to the chief and try to get feedback on how closely we can align with the current dept. logo or if we need to make additional changes besides text.

**Comment by icegulch** [comment]
Sounds great @skippy !! Like I said, it'd be pretty easy to place the florian cross within another badge shape, then maybe wrap text around that (as those other examples did). All of which is to say: 100% flexible!
